Packs/GitHub/Integrations/GitHub_IAM/GitHub_IAM.py

```diff
@@ -362,20 +363,20 @@ def main():
 
         elif command == 'iam-get-user':
             user_profile = get_user_command(client, args, mapper_in)
-            return_results(user_profile)
+            user_profile.return_outputs
```

Comment:
```suggestion
            user_profile.return_outputs()
```
please fix it in the following ones as well